portfolio/src/main/webapp/resources/errors/server-error.jsp

```diff
@@ -0,0 +1,9 @@
+<%@ page contentType = "text/html;charset=UTF-8" language = "java" %>
+<html>
+<head>
+    <title>Error Code 500</title>
```

Comment:
Since you're displaying this to a user that might not know what this means, I'd recommend `Status 500: Internal Server Error` to make it more clear.

Reply:
That is a great point, I will improve this